### PR TITLE
add pixnet provider

### PR DESCRIPTION
**Pull request by binotaliu**
This PR add a driver for pixnet (https://www.pixnet.net). Pixnet is a blog platform in Taiwan.

API Documentations: https://developer.pixnet.pro/

### CODE REVIEW

File: src/Pixnet/Provider.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SocialiteProviders\Pixnet;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```
Comment (atymic) [inline src/Pixnet/Provider.php:8]: Could we get a doc comment here with a link to the oauth docs please?

File: src/Pixnet/composer.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "authors": [
+        {
+            "name": "BinotaLIU",
+            "email": "me@binota.org"
+        }
+    ],
+    "require": {
+        "php": "^7.2",
+        "ext-json": "*",
+        "socialiteproviders/manager": "~2.0 || ~3.0"
```
Comment (atymic) [inline src/Pixnet/composer.json:14]: Can drop v2 here :)

**Review by atymic — COMMENTED** [review]
Just a couple of minor things, thanks for the provider!

**Comment by binotaliu** [comment]
Updated!

**Review by atymic — APPROVED** [review]

**Pull request merged**